build/codegen/mod.rs

```diff
@@ -9,6 +9,7 @@ pub mod stream;
 pub mod structs;
 
 #[derive(Clone)]
+#[allow(dead_code)]
```

Comment:
What's this lint for? The `UniqueIds` struct is used in a few different place

Reply:
I moved it to the anonymous field to make it more obvious